clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java

```diff
@@ -373,4 +517,62 @@ public void close() {
     public void onMemberEpochUpdated(Optional<Integer> memberEpochOpt, Optional<String> memberIdOpt) {
         memberIdOpt.ifPresent(s -> memberId = Uuid.fromString(s));
     }
+
+    /**
+     * Represents a request to acknowledge delivery that can be retried or aborted.
+     * ** UNDER CONSTRUCTION **
+     */
+    class AcknowledgeRequestState extends RequestState {
```

Comment:
Spotbug: As it not used yet, maybe can make class static or have a dummy usage for now to avoid build failure.